.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,64 @@
+---
+name: k6 Release 
+about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+title: 'k6 release v0.4x.0'
+---
+
+**Release Date**: 
+
+25th May 2023 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
+
+## Release Activities
+
+### At the beginning of the cycle
+
+- [ ] Create a new branch `release-notes-v04x` and add the new related release notes file.
+      
+All team members are expected to open PRs for adding new release notes items when they add a new change to the k6 code base. The issue's assignee should remember from time to time to do it, if some PRs are forgotten.
+```
+release-notes-v04x <--- my-single-feature-notes-branch
+```
+
+- [ ] Go through the potential [dependencies updates](https://github.com/grafana/k6/blob/master/Dependencies.md) then create a PR if any is relevant. 
```

Comment:
I don't mind putting this as the responsibility of the release driver, but we should probably agree inside the team first that we put this onto the release driver person's shoulder.

Reply:
Ok, let's use this PR to get the consensus. Are you (@grafana/k6-core) against moving here this responsibility?